doc/api/errors.md

```diff
@@ -599,182 +599,180 @@ found [here][online].
 
-Used when [`hash.digest()`][] is called multiple times. The `hash.digest()`
-method must be called no more than one time per instance of a `Hash` object.
+[`hash.digest()`][] was called multiple times. The `hash.digest()` method must
+be called no more than one time per instance of a `Hash` object.
 
 <a id="ERR_CRYPTO_HASH_UPDATE_FAILED"></a>
 ### ERR_CRYPTO_HASH_UPDATE_FAILED
 
-Used when [`hash.update()`][] fails for any reason. This should rarely, if
-ever, happen.
+[`hash.update()`][] failed for any reason. This should rarely, if ever, happen.
 
 <a id="ERR_CRYPTO_INVALID_DIGEST"></a>
 ### ERR_CRYPTO_INVALID_DIGEST
 
-Used when an invalid [crypto digest algorithm][] is specified.
+An invalid [crypto digest algorithm][] was specified.
 
 <a id="ERR_CRYPTO_INVALID_STATE"></a>
 ### ERR_CRYPTO_INVALID_STATE
 
-Used generically when a crypto method is used on an object that is in an
-invalid state. For instance, calling [`cipher.getAuthTag()`][] before calling
-`cipher.final()`.
+A crypto method was used on an object that is in an invalid state. For instance,
+calling [`cipher.getAuthTag()`][] before calling `cipher.final()`.
 
 <a id="ERR_CRYPTO_SIGN_KEY_REQUIRED"></a>
 ### ERR_CRYPTO_SIGN_KEY_REQUIRED
 
-Used when a signing `key` is not provided to the [`sign.sign()`][] method.
+A signing `key` was not provided to the [`sign.sign()`][] method.
 
 <a id="ERR_CRYPTO_TIMING_SAFE_EQUAL_LENGTH"></a>
 ### ERR_CRYPTO_TIMING_SAFE_EQUAL_LENGTH
 
-Used when calling [`crypto.timingSafeEqual()`][] with `Buffer`, `TypedArray`,
-or `DataView` arguments of different lengths.
+[`crypto.timingSafeEqual()`][] was called with `Buffer`, `TypedArray`, or
+`DataView` arguments of different lengths.
 
 <a id="ERR_DNS_SET_SERVERS_FAILED"></a>
 ### ERR_DNS_SET_SERVERS_FAILED
 
-Used when `c-ares` failed to set the DNS server.
+`c-ares` failed to set the DNS server.
 
 <a id="ERR_ENCODING_INVALID_ENCODED_DATA"></a>
 ### ERR_ENCODING_INVALID_ENCODED_DATA
 
-Used by the `util.TextDecoder()` API when the data provided is invalid
-according to the encoding provided.
+Data provided to `util.TextDecoder()` API is invalid according to the encoding
+provided.
 
 <a id="ERR_ENCODING_NOT_SUPPORTED"></a>
 ### ERR_ENCODING_NOT_SUPPORTED
 
-Used by the `util.TextDecoder()` API when the encoding provided is not one of
-the [WHATWG Supported Encodings][].
+Encoding provided to `util.TextDecoder()` API was not one of the
+[WHATWG Supported Encodings][].
 
 <a id="ERR_FALSY_VALUE_REJECTION"></a>
 ### ERR_FALSY_VALUE_REJECTION
 
-Used by the `util.callbackify()` API when a callbackified `Promise` is rejected
-with a falsy value (e.g. `null`).
+A `Promise` that was callbackified via `util.callbackify()` was rejected with a
+falsy value.
 
 <a id="ERR_HTTP_HEADERS_SENT"></a>
 ### ERR_HTTP_HEADERS_SENT
 
-Used when headers have already been sent and another attempt is made to add
-more headers.
+Headers have already been sent and another attempt was made to add more headers.
```

Comment:
`An attempt was made to add more headers when the headers had already been sent`, for consistency?